main.go

```diff
@@ -106,7 +106,7 @@ func main() {
 				updates.AddSuites(comment, suites)
 
 				if !dry {
-					client := testrail.NewClient("https://docker.testrail.com", username, token)
+					client := testrail.NewClient("https://mirantis.testrail.com", username, token)
```

Comment:
nit: having the URL be configurable would be useful for other projects that aren't using mirantis.testrail.com.

Reply:
Yeah if we did that we wouldn't have to be changing this right now. 

😞 
I think one misleading thing is that the github repo is public yet it's rigid, and kind of intended for one specific project.